tests/extmod/uasyncio_tcp_read_headers.py

```diff
@@ -0,0 +1,28 @@
+try:
```

Comment:
`# High-level test using asyncio to retrieve the HTTP headers from http://micropython.org`

Reply:
Yes, although this test will need to be moved to `tests/net_inet`